sonar-java-plugin/pom.xml

```diff
@@ -166,7 +166,7 @@
             <configuration>
               <shadedArtifactAttached>false</shadedArtifactAttached>
               <createDependencyReducedPom>false</createDependencyReducedPom>
-              <minimizeJar>true</minimizeJar>
+              <minimizeJar>false</minimizeJar>
```

Comment:
we should revert the flag to true and clean the filters: find the dependency with the Class that is not found and remove all the others that are not necessary but included transitively